docs/topics/rn-known-issues.adoc

```diff
@@ -18,21 +18,17 @@ For a complete list of all known issues, see the list of link:https://issues.red
 
 |link:https://issues.redhat.com/browse/WINDUP-3138[WINDUP-3138]
 |CodeReady Workspace plugin
-|The plugin does not work with CodeReady Workspace 2.10.
+|The plugin works with CodeReady Workspace 2.09, but not version 2.10.
 
-|link:https://issues.redhat.com/browse/WINDUP-3019[WINDUP-3019]
-|CLI
-|Bash completion feature displays an `mta-cli not found` error.
+|link:https://issues.redhat.com/browse/WINDUP-3165[WINDUP-3165]
+|Web console
+|MTA analysis fails on OpenShift Container Platform when web console custom labels are uploaded and advanced options are enabled.
 
-|link:https://issues.redhat.com/browse/WINDUP-3104[WINDUP-3104]
-|Web console on MacOS
-|After you run an analysis, if you click a report link, a "Not found" page is displayed.
+|link:https://issues.redhat.com/browse/WINDUP-3160[WINDUP-3160]
+|Web console
```

Comment:
I think I would change the component to "Web console on OpenShift etc" and remove "on OpenShift etc" from the description. Global.